.github/workflows/test-todo-to-pr-comment-issue.yml

```
@@ -0,0 +1,61 @@
+      - name: 📝 Inject test TODO fixture into PR diff
+        id: inject
+        if: github.event_name == 'pull_request'
+        run: |
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+          git config user.name "github-actions[bot]"
+          echo "original-sha=$(git rev-parse HEAD)" >> $GITHUB_OUTPUT
+          printf '// TODO: workflow test fixture\n' > todo-to-pr-comment-issue/src/test-fixture.ts
+          git add todo-to-pr-comment-issue/src/test-fixture.ts
+          git commit -m "test: TODO fixture [skip ci]"
+          git push
+      - uses: ./todo-to-pr-comment-issue
```

actionlint (1.7.12)

[error] file "./dist/index.mjs" does not exist in "/home/jailuser/git/todo-to-pr-comment-issue". it is specified at "main" key in "runs" section in "todo-to-pr-comment-issue" action

(action)